product_pack/models/product_product.py

```diff
@@ -51,10 +51,17 @@ def split_pack_products(self):
 
         return packs, (self - packs)
 
-    def price_compute(self, price_type, uom=False, currency=False, company=False):
+    def price_compute(
+        self, price_type, uom=False, currency=False, company=False, date=False
+    ):
+        company = company or self.env.company
+        date = date or fields.Date.context_today(self)
+
+        self = self.with_company(company)
```

Comment:
Is this necessary? I think it is included in the super method and it is not used after

Reply:
Yes, its included in the super method. The date is maybe required for the line 64?